### PR TITLE
TD-1305/automerge-gromit-option2

### PR DESCRIPTION
**Pull request by ermirizio**

### CODE REVIEW

File: policy/policy.go [hunk 1]
```diff
@@ -267,9 +267,26 @@ func (r *RepoPolicy) CreatePR(bundle, title, baseBranch string, dryRun bool) (st
 		}
 		log.Info().Str("baseBranch", baseBranch).
 			Str("title", title).Msg("calling CreatePR on github")
-		prURL, err = r.gitRepo.CreatePR(baseBranch, title, string(body))
+		pr, err := r.gitRepo.CreatePR(baseBranch, title, string(body))
 		if err != nil {
 			return "", err
+		} else {
```
Comment (alephnull) [inline policy/policy.go:273]: Is this else required? The return in the other branch ensures that control will never reach this point in the case of an error.

**Review by alephnull — CHANGES_REQUESTED** [review]
Thanks for this. It looks fine except for some minor style related things.

File: cmd/policy.go [hunk 2]
```diff
@@ -148,6 +148,7 @@ func init() {
 	policyCmd.PersistentFlags().StringVarP(&config.RepoURLPrefix, "prefix", "u", "https://github.com/TykTechnologies", "Prefix to derive the fqdn repo")
 	policyCmd.PersistentFlags().BoolVarP(&jsonOutput, "json", "j", false, "Output in JSON")
 	policyCmd.PersistentFlags().BoolVarP(&dryRun, "dry", "d", false, "Will not make any changes")
+	policyCmd.PersistentFlags().BoolVarP(&autoMerge, "auto", "a", false, "Will automerge if all requirements are meet")
```
[the remark this answers is not in the document]
Comment (ermirizio) [reply cmd/policy.go:151]: Felt the same

**Review by alephnull — APPROVED** [review]
Needs a rebase.

**Pull request merged**